extensions/ql-vscode/package.json

```diff
@@ -34,27 +34,8 @@
     }
   },
   "activationEvents": [
-    "onLanguage:ql",
     "onLanguage:ql-summary",
```

Comment:
Can this also be removed? It seems like this is also included in our `contributes.languages`.

Reply:
Yes indeed! In https://code.visualstudio.com/api/references/activation-events#onLanguage it's stated at some point. Thank you! 
It's 'interesting' that we activate the extension when the evaluator-log.summary is opened (why?..) by defining a 'language' that is connected to this file.

Reply:
It seems like we [do have some language support for summary files](https://github.com/github/vscode-codeql/blob/d0df2966c52e4f25124b59ca1a28717461c3ef14/extensions/ql-vscode/src/log-insights/summary-language-support.ts), so this is probably the easiest way to actually make VS Code use that language.